auto_submit/lib/helpers.dart

```diff
@@ -7,6 +7,9 @@ import 'dart:io';
 
 import 'package:shelf/shelf.dart';
 import 'package:shelf/shelf_io.dart';
+import 'package:logging/logging.dart';
+
+final Logger logger = Logger('helper');
```

Comment:
Let's move the loggers into their own file, like `service/log.dart` and instantiate it like `final Logger log = Logger('auto_submit')`

Changing the tag in every file will make it hard to read the logs and understand what was happening for a request